core/editor/src/tools/ellipse.rs

```diff
@@ -43,26 +43,43 @@ impl Fsm for EllipseToolFsmState {
 		match (self, event) {
 			(EllipseToolFsmState::Ready, Event::MouseDown(mouse_state)) if mouse_state.mouse_keys.contains(MouseKeys::LEFT) => {
 				data.drag_start = mouse_state.position;
+				operations.push(Operation::MountTempFolder { path: vec![] });
 				EllipseToolFsmState::LmbDown
 			}
+
 			(EllipseToolFsmState::Ready, Event::KeyDown(Key::KeyZ)) => {
 				if let Some(id) = document.root.list_layers().last() {
 					operations.push(Operation::DeleteLayer { path: vec![*id] })
 				}
```

Comment:
Let's call `mount` here, since it's the logical place we would mount the working layers.